tests/test_connector.py

```diff
@@ -65,12 +65,15 @@ def setUpClass(cls) -> None:
                                                   service_name='test')
         cls.connector_instance.run_consumers(names=('test1', 'test2'))
 
+    def test_01_not_null_service_id(self):
```

Comment:
Do these tests need to run in this order? easier to maintain without numbers if they don't, otherwise this is fine

Reply:
yeah it is needed as test 1 and test 2 are preconditions for running test 3

> Do these tests need to run in this order? easier to maintain without numbers if they don't, otherwise this is fine